### DISCUSSION TITLE
[Topics] Use dark mode specific images for applicable images

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14997] [Other Feature Feedback, Questions, & Ideas] [answered]
Some topics have images that don't contrast well in dark mode. A simple fix for this would allow a separate front matter tag for a dark mode specific logo.

Take the [iOS Topic](https://github.com/topics/ios) for example. The logo is black, and thus hard to see in dark mode. The front matter that this topic ([explore/topics/ios/index.md](https://github.com/github/explore/blob/main/topics/ios/index.md)) uses is `logo: ios.png`. In addition to this, it could have `logo-dark: ios-dark.png` for example.

**Comment by kenyonj** [comment, marked as answer]
Thanks for the feedback, @lx-is! I think this is a great idea. FWIW our current system doesn't support this, but we're always thinking of different ways that we can make the system better. Once we're able to prioritize our improvements for this repository (including topics and collections) and start working on implementing them, this will be on my list for sure.
